README.md

```diff
@@ -54,6 +54,7 @@ $ docker run -it -p 8080:8080 -e DEBUG="helia-http-gateway*" helia-http-gateway:
 | `TRUSTLESS_GATEWAYS` | Comma separated list of trusted gateways to fetch content from | [Defined in Helia](https://github.com/ipfs/helia/blob/main/packages/helia/src/block-brokers/trustless-gateway/index.ts) |
 | `USE_LIBP2P` | Whether to use libp2p networking | `true` |
```

Comment:
I wonder if we couldn't just merge the `USE_BITSWAP` and `USE_LIBP2P` vars to just be use `@helia/http` or use `helia`?

Reply:
I'm down with finagling the configs in another PR once we get verified-fetch in

Reply:
moved to https://github.com/ipfs/helia-http-gateway/issues/65